charts/metallb/templates/NOTES.txt

```diff
@@ -0,0 +1,4 @@
+MetalLB is now running in the cluster.
+
+Now you can configure it via its CRs. Please refer to the metallb official docs
```

Comment:
A link here wouldn't hurt

Reply:
Again this is smth that is relevant for the upstream